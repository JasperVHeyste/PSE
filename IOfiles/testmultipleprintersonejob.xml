<?xml version="1.0" encoding="UTF-8"?>
<SYSTEM>
	<DEVICE>
		<name>Office_Printer4</name>
		<emissions>3</emissions>
		<speed>40</speed>
		<cost>1</cost>
		<type>color</type>
	</DEVICE>
	<DEVICE>
		<name>Office_Printer3</name>
		<emissions>3</emissions>
		<speed>40</speed>
		<cost>1</cost>
		<type>scan</type>
	</DEVICE>
	<DEVICE>
		<name>Office_Printer2</name>
		<emissions>3</emissions>
		<speed>40</speed>
		<cost>1</cost>
		<type>bw</type>
	</DEVICE>
	<DEVICE>
		<name>Office_Printer1</name>
		<emissions>3</emissions>
		<speed>40</speed><cost>1</cost>
		<type>bw</type>
		<cost>1</cost>
	</DEVICE>
	<JOB>
		<jobNumber>1</jobNumber>
		<pageCount>2</pageCount>
		<userName>SergeDemeyer</userName>
		<type>bw</type>
	</JOB>
</SYSTEM>
