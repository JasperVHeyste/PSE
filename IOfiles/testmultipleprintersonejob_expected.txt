Printer 'Office_Printer2' finished black-and-white job:
    Number: 1
    Submitted by 'SergeDemeyer'
    2 pages
